docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install Git (64-bit).
+* Download and install Github Desktop (64-bit).
+* Download and install Miniconda(64-bit) for Python 2.7.
+* Download and install CMake.
+* Install DAYSIM version >= 4.0: Open Terminal and run the following commands
+
+  - ``git clone https://github.com/MITSustainableDesignLab/Daysim.git``
+  - ``mkdir build && cd build && cmake -DBUILD_HEADLESS=on -DCMAKE_INSTALL_PREFIX=$HOME/Daysim ../Daysim && make && make install``
+  - ``cp ./CMakeLists.txt /Daysim/CMakeLists.txt``

```

Comment:
this can never work

Reply:
Again, appears to have worked for me... although I might have gotten an error message when doing that 🤷‍♀

Reply:
ok. it's just, there _is_ no `./CMakeLists.txt`. I don't mean to bash your guide. I was just taking notes while following it and gave up here with the error message and I needed to go home. I will have another look at this soon and see if I can work out some stuff. Maybe we can even write it as a shell script to run. That would make things so much easier!